source/cms.c

```diff
@@ -20,6 +20,17 @@
 #define ENVELOPED_DATA_VERSION (2)
 #define ENVELOPED_DATA_RECIPIENT_VERSION (2)
 
+static inline int get_any_ber_asn1_element(
+    CBS *cbs,
+    CBS *out,
+    CBS_ASN1_TAG *out_tag,
+    size_t *out_header_len) {
+
+    int indefinite_temp;
+
+    return CBS_get_any_ber_asn1_element(cbs, out, out_tag, out_header_len, NULL, &indefinite_temp);
```

Comment:
Are we making use of indefinite-length elements or can we use `CBS_get_any_asn1_element` instead of `CBS_get_any_ber_asn1_element`? If we make use of indefinite elements, we should likely check `out_indefinite` to distinguish indefinite from empty elements.

Reply:
Our code uses indefinite elements and doesn't need that flag at all. I don't prefer changing the code's behavior now but we, for sure, will utilize that flag when we refactor the code. @mariusknaust